ONSdigital/census-rm-action-scheduler
# Case history updates

# Motivation and Context

The case history should show an event when a case is included in a print file. We need to send events back to the Case Processor service from the Action Scheduler when an action rule is triggered, so that we can record an event against the cases which were selected.

# What has changed



Created DTO for message sent back to Case. Created new queue and exchange. Created builder for message. Build and send message when we send message onto Print Service.

# How to test?



Load a sample. Create an action plan & rule. You should see messages on the `action.events` queue appear when the rule executes.

# Links



Trello: https://trello.com/c/seWLHMKQ

# Screenshots (if appropriate):
